src/plots/cartesian/graph_interact.js

```diff
@@ -1998,14 +2023,18 @@ fx.dragElement = function(options) {
         dragCover.onmouseup = onDone;
         dragCover.onmouseout = onDone;
 
+        dragCover.style.cursor = window.getComputedStyle(options.element).cursor;
```

Comment:
also useful for zoom and pan as well as select, so the cursor doesn't go back to pointer when you start the action.